builder/module-config.nix

```diff
@@ -0,0 +1,169 @@
+{ config, pkgs, lib, ... }:
+let
+  inherit (import ../files.nix {
+    inherit pkgs;
+    inherit (config) release feedsSha256;
+    inherit (config.hardware) target;
+    variant = config.hardware.subtarget;
```

Comment:
Shouldn't we name variant subtarget here, too or at least add a TODO comment to rename it?

Reply:
I didn't want this PR to turn into a whole repository rewrite. But I'll add a TODO.